Makefile

```diff
@@ -13,15 +13,9 @@ build:
 	GOOS=windows GOARCH=amd64 go build -o ls-lint-windows.exe
 
 build-npm:
```

Comment:
referending to my [comment](https://github.com/loeffel-io/ls-lint/pull/11#issuecomment-598664338) i would love to have separated make commands for every platform. 

should look like:

```
build-npm:
	make build-npm-darwin
	make build-npm-linux
	make build-npm-windows

build-npm-darwin:
	cp ls-lint-darwin npm/bin/ls-lint-darwin
	chmod +x npm/bin/ls-lint-darwin

build-npm-linux:
	cp ls-lint-linux npm/bin/ls-lint-linux
	chmod +x npm/bin/ls-lint-linux

build-npm-windows:
	cp ls-lint-windows.exe npm/bin/ls-lint-windows.exe
	chmod +x npm/bin/ls-lint-windows.exe
```